src/utils/offers/offerAggregateOptions.js

```
@@ -50,11 +50,15 @@ const offerAggregateOptions = (query, params) => {
   }
 
   if (proficiencyLevel) {
-    match.proficiencyLevel = { $in: proficiencyLevel }
+    const decodedValue = decodeURIComponent(proficiencyLevel).split(',')
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)